tests/library/Respect/Validation/Rules/DateTest.php

```diff
@@ -72,5 +72,20 @@ public function testDateTimeExceptionalFormatsThatShouldBeValid()
         $this->dateValidator = new Date('r');
         $this->assertTrue($this->dateValidator->assert('Thu, 29 Dec 2005 01:02:03 +0000'));
     }
+
+    public function testDateTimeSystemTimezoneIndependent()
+    {
+        date_default_timezone_set('UTC');
+        $this->dateValidator = new Date('c');
+        $this->assertTrue($this->dateValidator->assert('2005-12-30T01:02:03+01:00'));
+        $this->dateValidator = new Date('c');
+        $this->assertTrue($this->dateValidator->assert('2004-02-12T16:19:21+01:00'));
+        date_default_timezone_set('Europe/Amsterdam');
+        $this->dateValidator = new Date('c');
+        $this->assertTrue($this->dateValidator->assert('2005-12-30T01:02:03+01:00'));
+        $this->dateValidator = new Date('c');
+        $this->assertTrue($this->dateValidator->assert('2004-02-12T16:19:21+01:00'));
```

Comment:
@henriquemoody I'm not totally sure how an assertFalse would make it more clear what I fixed? Maybe little pointer in the right direction? Dataprovider I will implement now... :)